wiki-docs/docs/interfaces/erc20/native-erc20.md

```diff
@@ -0,0 +1,83 @@
+The `INativeERC20` interface represents a native ERC20 token on the L2 side of the childchain. It allows the minting and burning of tokens under the control of a predicate address. This user guide will explain how to interact with the functions provided by the `INativeERC20` interface.
+
+## Functions
+
+### initialize()
+
+This function sets the values for the predicate, rootToken, name, symbol, and decimals.
+
+#### Parameters
+
+- `predicate_` (address): The address of the predicate controlling the child token.
+- `rootToken_` (address): The address of the root token on the mainchain.
+- `name_` (string): The token's name.
+- `symbol_` (string): The token's symbol.
+- `decimals_` (uint8): The number of decimals for the token.
```

Comment:
`initialize` function has one more parameter: `uint256 tokenSupply_` which represents initial total supply of given token.

Reply:
added